### PR TITLE
🚇 Release 1.0.1

### PR DESCRIPTION
**Pull request by hassan-pullflow**
### Summary of Changes 

- #3 
- #4
- #5 
- #6 
- #7  
- #8

### CODE REVIEW

File: src/views/webviews/welcome/welcome.ts [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+              of your active pull requests. It enables quick identification and focus on pull requests needing attention,
+              all within VS Code Quick Pick.</p>
+            <img src="${assetsPath}/pr-dashboard.png" id="pr-dashboard" alt="PR-Dashboard" />
+          </div>
+          <div>
+            <h3>Quick Actions</h3>
+            <p>The extension comes with a set of actions designed to help you take care of your code review tasks and return
+              to your flow state. Actions you can take on pull request from VS Code are: </p>
+            <ul>
+              <li>&nbsp;&nbspOpen in Slack</li>
```
Comment (zakmandhro) [inline src/views/webviews/welcome/welcome.ts:85]: Missing ; after nbsp

File: src/views/webviews/welcome/welcome.ts [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+                In </a>
+            </p>
+          </div>
+          <div>
+            <img src="${assetsPath}/main.png" id="main-img" alt="Main" />
+          </div>
+          <div>
+            <h3>Supercharge your workflow with Pullflow for VS Code</h3>
+            <ul>
+              <p>●&nbsp;&nbsp;&nbsp;Effortlessly communicate across GitHub, Slack, and VS Code.</p>
```
Comment (zakmandhro) [inline src/views/webviews/welcome/welcome.ts:65]: why is this a <p> and not <li> like the one below?

File: src/views/webviews/welcome/welcome.ts [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+              your own features.</p>
+            <p>This is just the beginning of Pullflow's VS Code Slack integration, and we are eager to make improvements
+              based on your feedback. Please don't hesitate to share your thoughts with us through <a
+                href="https://github.com/gradience-io/kana/issues">GitHub issues</a> or on Twitter <a
+                href="https://twitter.com/pullflow">@pullflow</a>.</p>
+          </div>
+        </section>
+      </section>
+      <footer id="footer">
+        <i>This tool is designed by IDE-integration team at Pullflow for developers seeking a unified workspace that reduces
```
Comment (zakmandhro) [inline src/views/webviews/welcome/welcome.ts:157]: This is a run-on sentence and redundant. Consider revising.

File: src/views/webviews/welcome/welcome.ts [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+          </div>
+          <div id="command-div">
+            <h3>Commands</h3>
+            <table>
+              <tbody>
+                <tr>
+                  <td>Pullflow: Active Pull Requests (Ctrl/Cmd+Shift+,)</td>
+                </tr>
+                <tr>
+                  <td>Pullflow: Login</td>
```
Comment (zakmandhro) [inline src/views/webviews/welcome/welcome.ts:122]: We use 'sign in' (not login) in most places. Let's make it consistently 'Sign in'.

File: src/views/webviews/welcome/welcome.ts [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+
+          <div id="intro">
+            <p> Pullflow is an AI-enhanced code review collaboration platform backed by the co-founder of GitHub and used by
+              some of the most innovative dev teams, including <strong> Epic Games, Avenue, Hear.com,
+                and RedwoodJS. </strong> </p>
+            <p>
+              It offers seamless integration with your <strong>Slack</strong> and <strong>GitHub </strong> accounts. Simply
+              authenticate your accounts within Pullflow, and the
+              synchronization will be automatically established. New to Pullflow? <a href="https://app.pullflow.com"> Sign
+                In </a>
```
Comment (zakmandhro) [inline src/views/webviews/welcome/welcome.ts:56]:
```suggestion
              ```suggestion
  Sign in to your accounts with Pullflow to enable two-sync between GitHub and Slack. New to Pullflow? <a href="https://pullflow.com">Sign up for free.</a>
```

File: styles/welcome.css [hunk 6]
```diff
@@ -0,0 +1,92 @@
+body {
+  font-family: Arial, sans-serif;
```
Comment (zakmandhro) [inline styles/welcome.css:2]: Why Arial? I don't think we should specify font. Have we modeled this after some other extension HTML?

File: styles/welcome.css [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+.column:first-child {
+  flex: 2;
+}
+
+.column:last-child {
+  margin-left: 50px;
+}
+
+#desc {
+  color: #f6f4f9;
```
Comment (zakmandhro) [inline styles/welcome.css:33]: Will these colors work in light and dark?

**Review by zakmandhro — CHANGES_REQUESTED** [review]
Several cosmetic changes and textual improvements.

File: src/views/webviews/welcome/welcome.ts (hunk 4 above)
Comment (amna-pullflow) [reply src/views/webviews/welcome/welcome.ts:120]: @hassan-pullflow this is still pending

**Review by srzainab — APPROVED** [review]

**Review by amna-pullflow — APPROVED** [review]

**Review by zakmandhro — APPROVED** [review]

**Pull request merged**